lib/library.rb

```diff
@@ -0,0 +1,57 @@
+require './lib/book'
+require './lib/author'
+
+class Library
+
+    attr_reader :name, :books, :authors
+
+    def initialize(name, books = [], authors = [])
```

Comment:
I suppose you could do this, but we didn't ask for books and authors to be passed in as arguments. Generally speaking, we'd want to limit the amount of arguments since it can create too much complexity.